README.md

```diff
@@ -1,72 +1,38 @@
-
-* .NET Core contributors.
-* Linux distribution maintainers.
-* Cloud service developers.
-
-You do not have to build the entire product to contribute to .NET Core. Often, you only need to build a single binary to test a change. There are some scenarios where building the whole product is useful, such as adding and testing a feature that requires changes to multiple repos.
+## Goals
 
-## What the Scripts Do
+The key goal of this repository is to satisfy the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html). Many Linux distributions have similar rules. These rules tend to have two main principles: consistent reproducibility, and source code for everything.
 
-The scripts can be thought of as solving challenges that would otherwise making building the whole product difficult. The following challenges are the primary ones that developers often hit before these scripts were available.
+A secondary goal of source-build is to allow .NET Core contributors to build a .NET Core SDK with coordinated changes in multiple repositories. However, the developer experience is significantly better in individual repositories and, if possible, contributors should make and test changes in the target repo, not source-build.
 
-* .NET Core is composed of several repositories that all need to be built.
-* The .NET Core SDK generated by the build requires a specific layout in order to correctly function.
-* Most of the product is written in managed code and requires the .NET Core SDK to build. This approach is a great use of the product, but presents a boot-strapping problem for the build.
+## What does the source-build infrastructure do?
 
-## Goals
+Source-build solves common challenges that most developers encounter when trying to build the whole .NET Core SDK from source.
 
-Many Linux distributions have specific rules for official packages. The rules can be summarized as two main rules: source for everything, and consistent reproducibility.
+* .NET Core is composed of many repositories that need to be built at a specific combination of commits.
+* Each repository's build output needs to flow into the next repository's build.
+* Nearly all .NET Core repositories require the .NET Core SDK to build. This is a circular dependency, which presents a bootstrapping problem.
 
-A key goal of this repository was to satisfy the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html).
+The source-build repository contains scripts and build logic to help Linux distribution maintainers address these challenges.
 
 ## License
 
-This repo is licensed with [MIT](LICENSE.txt).
+.NET (including the source-build repo) is licensed under the [MIT](LICENSE.txt) license.
```

Comment:
I dont think this is entirely accurate. [ASP.NET Core 3.1 is under Apache 2.0](https://github.com/dotnet/aspnetcore/tree/release/3.1). There's also [third party licenses there](https://github.com/dotnet/aspnetcore/blob/release/3.1/THIRD-PARTY-NOTICES.txt).

Reply:
Interesting, should probably raise this at dotnet/runtime too: https://github.com/dotnet/runtime/blob/eef01694a727a20cf1c0999922a164caa728c466/README.md#L66. I'll reduce this to just the source-build repo.

Reply:
It looks like the ".NET (including..." phrasing came from the very first CoreCLR commit: https://github.com/dotnet/coreclr/blame/2f8d015389c8408dc042991b5df7812ecdd892fe/README.md#L50. I've reverted this PR back to basically what it was before ("This repo"), I'll leave it up to you if this is worth following up on in dotnet/runtime. Maybe this is a relic of ".NET Core" and "ASP.NET [Core? 5?]" being more distinct.